drivers/netif-driver.c

```diff
@@ -146,6 +149,12 @@ int create_netif(char *conf)
 	if (!is_ppp) {
 		err = netifapi_netif_add(ni, NULL, NULL, NULL,
 			(char *)storage + sz, netif_dev_init, tcpip_input);
+#if LWIP_IPV6
+		if (err == ERR_OK) {
+			netif_set_ip6_autoconfig_enabled(ni, 1);
```

Comment:
needs to be under `LWIP_IPV6_AUTOCONFIG`, otherwise compilation would fail

Reply:
done